broker-cli/scripts/postinstall.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+# Setup auto-completion for bash
+echo "source <(sparkswap completion bash)" >> ~/.bashrc && source ~/.bashrc
```

Comment:
do we need to check that `bashrc` checks first? Or is there another way to determine the default shell?

I'd hate to populate a `.bashrc` if the user doesn't use `bash`.